test/specs/WebexMeeting/WebexMeetingPage.js

```diff
@@ -0,0 +1,95 @@
+import {createTestUser, removeTestUser} from '@webex/test-users';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+class WebexMeetingPage {
+  constructor() {
+    this.user = null;
+  }
+
+  setUser(user) {
+    this.user = user;
+  }
+
+  get accessTokenInput() {
```

Comment:
can we make these one-liners like in the webdriver.io examples? And drop the "Input" suffix.